cmd/consumer/consumer.go

```diff
@@ -85,14 +90,24 @@ func run() error {
 		KafkaLogErrors:    true,
 	}
 	consumer := consumer.NewKafkaConsumer(input)
+	consumerInitError := make(chan struct{})
```

Comment:
Are we supposed to pass some error into this channel in goroutine?

Reply:
I was trying something else and figured it's not needed. Forgot to remove all the related code.